src/Roadmap.js

```diff
@@ -209,7 +236,7 @@ const Roadmap = ({ identifier, onClose }) => {
           background={editing ? { color: 'background-contrast' } : undefined}
           gap="small"
         >
-          <Header background={{ color: 'background', dark: true }} pad="small">
+          <Header background={{ color: 'background-contrast' }} pad="small">
             <Button icon={<Navigate />} onClick={onClose} />
             <Heading textAlign="center" size="24px" margin="none">
```

Comment:
Should I go through and update anywhere in the old code that uses set px values to use t-shirt sizing instead?

Reply:
yes